### PR TITLE
[common] Add some methods to RoaringBitmap32

### PR DESCRIPTION
**Pull request by hang8929201**
### Purpose

Add and/or operation to RoaringBitmap32.

### CODE REVIEW

**Comment by hang8929201** [comment]
add more methods

File: paimon-common/src/main/java/org/apache/paimon/utils/RoaringBitmap32.java [hunk 1]
```diff
@@ -39,6 +40,10 @@ public RoaringBitmap32() {
         this.roaringBitmap = new RoaringBitmap();
     }
 
+    public RoaringBitmap32(RoaringBitmap roaringBitmap) {
```
Comment (JingsongLi) [inline paimon-common/src/main/java/org/apache/paimon/utils/RoaringBitmap32.java:43]: We should not support this, `RoaringBitmap` should be hided.

Comment (hang8929201) [reply paimon-common/src/main/java/org/apache/paimon/utils/RoaringBitmap32.java]: okay, can I just change it to private?

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**